### DISCUSSION TITLE
Stuck at Enrty_with_update.py

### DISCUSSION THREAD

**Discussion by djaiswal21** [lllyasviel/Fooocus#2669] [Q&A] [answered]
Hello all, 
This is my first time using Fooocus, 

I went over all the step and see the expected results. however when I try running the python entry_with_update.py command I get stuck here : 
![image](https://github.com/lllyasviel/Fooocus/assets/31632896/b15d5e16-fe59-421c-a3da-efceb705a12d)

I have this screen for over 15 mins now. 

I even tried downloading the .pth file and added it under the models/ vae_approx/ folder but still no progress. 

Any hints or help to move forward would be appreciated

Thanks

**Comment by mashb1t** [comment, marked as answer]
Please try to manually download the file, put it in the given target directory and restart Fooocus.

**Reply by djaiswal21** [reply]
is there a list of all the files what we need ? 
I already did xlvaeapp.pth, 
Thanks

**Reply by mashb1t** [reply]
Not a whole list but please check the code near the end of the config.py file. Disabling VPN and adjusting network settings of your firewall may also help you to fix the download.
